plugins/obs-ffmpeg/ffmpeg-mux/ffmpeg-mux.c

```diff
@@ -258,13 +257,13 @@ static void ffmpeg_log_callback(void *param, int level, const char *format,
 			out.array, ANSI_COLOR_RESET);
 		fflush(stderr);
 	}
-
-	dstr_free(&out);
 #else
-	UNUSED_PARAMETER(level);
-	UNUSED_PARAMETER(format);
-	UNUSED_PARAMETER(args);
+	if (level == AV_LOG_ERROR) {
+		fprintf(stderr, "%s", out.array);
+		fflush(stderr);
+	}
```

Comment:
If ffmpeg logs `AV_LOG_ERROR` for non-fatal errors then our `stderr` will fill up and block since OBS doesn't read it unless there's a fatal error. Are we sure all paths that his this are fatal errors?

Reply:
right; i'll put a condition to identify that this is the log entry targeted from librist; if it is i'll pass to sdterr; if not, I won't.